src/utils/general-helpers.ts

```diff
@@ -40,8 +40,9 @@ export const getIdentifierFromId = (id: string) => id.split('/').pop() ?? '';
 
 export const getInitials = (name: string) => {
   if (!name) return '';
-
-  const splittedNames = name.split(' ');
+  const cleanedName = name.trim().replace(/\s+/g, ' '); // Ensure single spaces between names
+  if (!cleanedName) return ''; // Handle when name is just a string of spaces
+  const splittedNames = cleanedName.split(' ');
```

Comment:
Synes kanskje det er litt unødvendig med disse kommentarene, da koden i seg selv er ganske selvforklarende😬 Tenker det kan være greit om vi i utgangspunktet ikke legger inn kommentarer selv, men heller på oppfordring av andre utviklere. Så om man ikke forstår koden uten kommentarer, kan man si ifra om det

```suggestion
  const cleanedName = name.trim().replace(/\s+/g, ' '); 
  if (!cleanedName) return ''; 
  const splittedNames = cleanedName.split(' ');
```